### DISCUSSION TITLE
Add an option to disable the small album art thumbnail next to songs when viewing an album

### DISCUSSION THREAD

**Discussion by RamiroRincon** [jeffvli/feishin#646] [Ideas]
When viewing an album, you get a preview of the album art for every single song. I see this useful for a playlist, where songs may come from different albums and it's helpful to get a quick look at which album a song belongs to. In an actual album though, it's redundant.

I think, in that place there should be a play/pause button or a playing/stopped indicator. Something similar to other players like Spotify. Or just nothing there at all.

**Comment by jeffvli** [comment]
This is already possible. Just remove the `Title (Combined)` column and add `Title` instead.

![image](https://github.com/user-attachments/assets/8a8d5743-d88f-41bb-b8c6-378f92101333)
